src/server/scripts/Outland/CoilfangReservoir/SerpentShrine/boss_lurker_below.cpp

```diff
@@ -213,7 +216,7 @@ class go_strange_pool : public GameObjectScript
             if (roll_chance_i(instance->GetBossState(DATA_THE_LURKER_BELOW) != DONE ? 25 : 0) && !instance->IsEncounterInProgress())
             {
                 player->CastSpell(player, SPELL_LURKER_SPAWN_TRIGGER, true);
-                if (Creature* lurker = ObjectAccessor::GetCreature(*go, instance->GetGuidData(NPC_THE_LURKER_BELOW)))
+                if (Creature* lurker = go->SummonCreature(NPC_THE_LURKER_BELOW, 40.4058f, -417.108f, -21.5911f, 3.03312f, TEMPSUMMON_CORPSE_TIMED_DESPAWN, 604800000))
```

Comment:
source on despawn timer?

Reply:
One week time

Reply:
Or is it 0?

Reply:
I'd say just make it permanent. It resets on weekly wipe anyway. And it despawns on evade so

Reply:
TEMPSUMMON_MANUAL_DESPAWN should work